FakeItEasy-NetCore.sln

```diff
@@ -0,0 +1,225 @@
+		{42A92357-2A87-4B87-A591-CA78B6CE4461}.Release|Any CPU.Build.0 = Release|Any CPU
+		{74165807-676F-4464-B607-8770C13E8991}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{74165807-676F-4464-B607-8770C13E8991}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{74165807-676F-4464-B607-8770C13E8991}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{74165807-676F-4464-B607-8770C13E8991}.Release|Any CPU.Build.0 = Release|Any CPU
+		{CF137AE3-7C4F-41BD-A4AD-762BBED2255E}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{CF137AE3-7C4F-41BD-A4AD-762BBED2255E}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{CF137AE3-7C4F-41BD-A4AD-762BBED2255E}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{CF137AE3-7C4F-41BD-A4AD-762BBED2255E}.Release|Any CPU.Build.0 = Release|Any CPU
+		{AA7CF6E8-C3F2-43CC-A765-5B05AADED390}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{AA7CF6E8-C3F2-43CC-A765-5B05AADED390}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{AA7CF6E8-C3F2-43CC-A765-5B05AADED390}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{AA7CF6E8-C3F2-43CC-A765-5B05AADED390}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(NestedProjects) = preSolution
+		{1ED2F1A2-40E3-498C-B13F-AF066EACF58F} = {2EEDF4EB-ED06-4370-9C0A-59502DF1B7F6}
+		{0E26272D-045A-47C2-8928-BD9B980115D5} = {DC5EAE30-96E3-4D38-B3C5-33E587C9FB76}
+		{C291B61B-0729-4DC5-B8E4-147A8E3F801F} = {5D41D463-D53B-4ADC-90AD-44AA03D6340D}
+		{69A27B0A-710B-4CEC-B159-56E34C788665} = {5D41D463-D53B-4ADC-90AD-44AA03D6340D}
+		{193C944B-733B-4E01-87A3-73F7AB40CDC7} = {2EEDF4EB-ED06-4370-9C0A-59502DF1B7F6}
+		{FA539A2D-98DC-4300-9A2C-EC6FE38A7392} = {575A3C3B-17BA-4A8A-9F53-05320913D0A6}
+		{80721425-68E5-48DC-87EA-41D63D0B45FA} = {59466C4D-C407-4DD9-B77B-D99162B9FC7C}
+		{30396678-D653-4C67-A507-1749424A24CD} = {5D41D463-D53B-4ADC-90AD-44AA03D6340D}
+		{DC5EAE30-96E3-4D38-B3C5-33E587C9FB76} = {AD790E7D-D664-4B5F-8BE8-01507A2A9F3B}
+		{5D41D463-D53B-4ADC-90AD-44AA03D6340D} = {AD790E7D-D664-4B5F-8BE8-01507A2A9F3B}
+		{575A3C3B-17BA-4A8A-9F53-05320913D0A6} = {AD790E7D-D664-4B5F-8BE8-01507A2A9F3B}
+		{5A081DFC-0269-480B-853A-89D09AF478A8} = {072853A5-377B-47C4-8644-AF1F4FD669E2}
+		{F97E3B4B-080D-414A-9F00-EE20F4451E36} = {59466C4D-C407-4DD9-B77B-D99162B9FC7C}
+		{38F3191B-1CE0-4EE0-930F-794A2F2F4CDB} = {DC5EAE30-96E3-4D38-B3C5-33E587C9FB76}
+		{9D4A41F1-D444-4244-AAAC-F610D9977B2F} = {AD790E7D-D664-4B5F-8BE8-01507A2A9F3B}
+		{42A92357-2A87-4B87-A591-CA78B6CE4461} = {9D4A41F1-D444-4244-AAAC-F610D9977B2F}
+		{74165807-676F-4464-B607-8770C13E8991} = {59466C4D-C407-4DD9-B77B-D99162B9FC7C}
+		{CF137AE3-7C4F-41BD-A4AD-762BBED2255E} = {59466C4D-C407-4DD9-B77B-D99162B9FC7C}
+		{AA7CF6E8-C3F2-43CC-A765-5B05AADED390} = {DC5EAE30-96E3-4D38-B3C5-33E587C9FB76}
+	EndGlobalSection
+	GlobalSection(MonoDevelopProperties) = preSolution
+		version = 0.1
+		StartupItem = src\FakeItEasy\FakeItEasy.csproj
+	EndGlobalSection
+EndGlobal

```

Comment:
I eventually got something to work, but needed to do more than the instructions, as I had already installed those components. I ended up running a few `dotnet` commands in sequence, including `run` and `build` before VS would build that solution for me. I don't think MSBuild builds it yet, but a `dotnet build` and `dotnet test` ended up running the tests (all passed!) for me…